content/shmem_broadcast.tex

```diff
@@ -72,12 +72,15 @@
     needed to ensure this): The \VAR{pSync} array on all \ac{PE}s in the
     \activeset{} is not still in use from a prior call to a broadcast routine.  The
     \dest{} array on all \ac{PE}s in the \activeset{} is ready to accept the
-    broadcast data.
+    broadcast data. \newtext{The data in the \source{} array on the root \ac{PE} is ready to be broadcasted.}
```

Comment:
@jdinan Even without this new change, we had to check the dest buffer for readiness, before you send the data from root to non-root PEs. 

IIRC, the suggestion was to use something like this - ““The shmem_broadcast cannot start unless the root initiates .. “ The issue with this is that the dest buffer still has to checked for readiness. Further, we will essentially be restricting the implementations to not use “Gets”.